### ISSUE TITLE
 two-stage framework

### ISSUE THREAD

**Issue by BudBudding** [kellyiss/SituFormer#2]
With the two-stage framework, can I just do the second stage of argument detection if I already have the action recognition results corresponding to each image? What should I do?

**Comment by kellyiss** [comment]
Absolutely. The input to the second stage consists of the label pf action and its corresponding arguments, which are decoupled from the first stage model.
